src/meta/src/rpc/metrics.rs

```diff
@@ -486,7 +499,15 @@ impl MetaMetrics {
         let table_info = register_int_gauge_vec_with_registry!(
             "table_info",
             "Mapping from table id to (actor id, table name)",
-            &["table_id", "actor_id", "table_name"],
+            &["materialized_view_id", "table_id", "actor_id", "table_name"],
+            registry
+        )
+        .unwrap();
+
+        let mv_info = register_int_gauge_vec_with_registry!(
+            "materialized_info",
+            "Mapping from materialized view id to (table id, table name)",
+            &["materialized_view_id", "table_id", "table_name"],
```

Comment:
Will sink be included in this metric? If yes, I think it is better to just use `id` instead of the `materialized_view_id` as the label name.